tests/mocha/serializer_test.js

```diff
@@ -0,0 +1,1757 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+// TODO: Move this into samples as part of the dev-tools package.
+/**
+ * Constructs a serializer test.
+ * @param {string} title The title of this testcase.
+ * @param {string} xml The XML to use for the round-trip test.
+ * @constructor
+ * @extends {TestCase}
```

Comment:
Correction on my previous comment, based on my reading, I think this is supposed to be `@implements`. The other one needs to be `@exports` because there is a generic template type involved.